topics/controllers.js

```diff
@@ -0,0 +1,14 @@
+const {selectTopics} = require('./models')
+
+
+exports.getTopics = (request, response, next) => {
+
+    selectTopics().then(topics => {
+      response.status(200).send(topics)
+    })
+    .catch(err => {
+      // console.log(err, '<<< controllers')
```

Comment:
Don't forget to remove any comments from your code. Keep it looking nice and tidy 😄